src/components/VCSMenu.tsx

```diff
@@ -80,22 +87,115 @@ export class VCSMenu extends React.Component<VCSMenuProps, VCSMenuState> {
     this.updateSelectedVariables = this.updateSelectedVariables.bind(this);
   }
 
+  async resetState() {
+    console.log("VCS Menu state reset.");
+    this.graphicsMenuRef.resetGraphicsState();
+    this.varMenuRef.setSelected(new Array<Variable>());
+    this.setState({
+      file_path: "",
+      plotReady: false,
+      loadedVariables: new Array<Variable>(),
+      selectedVariables: new Array<Variable>(),
+      selected_gm_group: "",
+      selected_gm: "",
+      selected_template: ""
+    });
+  }
+
   update(vars: Array<string>, gms: Array<any>, templates: Array<any>) {
     console.log(vars, gms, templates);
   }
 
+  getGraphicsSelections() {
+    // Load the selected graphics method from meta data (if exists)
+    let gm_data: [string, string] = notebook_utils.getMetaDataNow(
+      this.state.notebook_panel,
+      GRAPHICS_METHOD_KEY
+    );
+
+    // If the data is not null, set the selected graphic method and group
+    if (gm_data) {
+      this.setState({
+        selected_gm: gm_data[0],
+        selected_gm_group: gm_data[1]
+      });
+      this.graphicsMenuRef.setState({
+        selectedGroup: gm_data[0],
+        selectedMethod: gm_data[1],
+        tempGroup: gm_data[0]
+      });
+      console.log("Graphics selection loaded.");
+    } else {
+      // No meta data means fresh notebook, reset the graphics
+      this.graphicsMenuRef.resetGraphicsState();
+    }
+  }
+  async copyGraphicsMethod(
+    groupName: string,
+    methodName: string,
+    newName: string
+  ) {
+    try {
+      //Check that the method doesn't already exist in the selected group
+      if (this.props.getGraphicsList()[groupName].indexOf(newName) < 0) {
```

Comment:
this is another case where inverting the if/else can cut out a set of brackets and indentation.